bundles/core/src/main/java/com/adobe/cq/wcm/core/components/models/Carousel.java

```diff
@@ -39,6 +39,13 @@
      */
     String PN_DELAY = "delay";
 
+    /**
+     * Name of the resource property that indicates whether automatic pause on hovering the carousel is disabled, or not.
+     *
+     * @since com.adobe.cq.wcm.core.components.models 12.6.0
+     */
+    String PN_AUTOPAUSE_DISABLED = "autopauseDisabled";
```

Comment:
How about renaming the property `autopause` and invert its values?

Reply:
Because we generally want the defaults to be `false`

Reply:
Thx for refreshing my memory! Yes I agree that defaults should be unchecked, even if it makes property names sometimes a bit heavy.